src/components/Calculator.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+
+export default class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
```

Comment:
- [ ] Since you are adding props, you should validate it.

Reply:
Thanks for the review. decided to remove the props since I am not using them in this case.